data/json/construction.json

```diff
@@ -1961,7 +1961,7 @@
     "category": "FURN",
     "required_skills": [ [ "survival", 0 ] ],
     "time": "1 m",
-    "components": [ [ [ "withered", 50 ] ] ],
+    "components": [ [ [ "withered", 5 ] ] ],
```

Comment:
This is like 100g of dried plant material, that does not seem particularly bed-sized to me.  Similarly the volume is 250ml per unit, some figuring comparing this to a twin mattress sugggests that you're looking at more like 250 - 300 L for a workable volume, suggesting that if anything, the 50 units of withered plants is generous.

Reply:
This is supposed to be a pile of leaves, used only for sleeping in an emergency when you can't have anything else. I do agree that it is not particularly realistic, we would need actual leaves for that since a big number of leaves are easier to obtain than withered plants.
Even looking at it from a balance perspective: There is not a reason to construct a pile of leaves because it needs so many withered plants and is not even particularly good. But maybe the straw bed needs to require more withered plants, too.

Reply:
Sure thing, the straw bed should total 100-200L of material too.

Reply:
> There is no reason to construct a pile of leaves because it needs so many withered plants and is not even particularly good

It's vanishingly rare that anything craftable is ever removed from CDDA, but perhaps this is one to consider?